infrastructure/main.tf

```diff
@@ -40,7 +42,7 @@ resource "azurerm_resource_group" "rg" {
 }
 
 resource "azurerm_storage_account" "provider" {
-  name                      = "bulkscanning${var.env}"
+  name                      = "${substr(local.account_name, 0, 24)}"
```

Comment:
I think it would be best to see an error when the name is too long. Otherwise, we risk having environment component taken out of the name.

Reply:
this will be only for preview I believe so yes it would be good if we can do this only for preview environment and not trim for any other.